core/typescript/src/lib/metadata.ts

```diff
@@ -99,6 +108,17 @@ export interface ServiceType
   readonly deployment: string;
 }
 
+/**
+ * Interface that represents a typing classification for relations.
+ *
+ * @group For both project creators and package authors
+ */
+export interface RelationType
+  extends Connectable,
+    OptionalLabelAware,
+    OptionalIconAware,
+    OptionalDescAware {}
+
```

Comment:
This is again from the latest discussion around diagram tooling where each line connecting a service/component with another service/company needs to have more info such as label/description/icon similarly to how it's been set up with each component/service type.